app/models/nic/base.rb

```diff
@@ -318,14 +318,14 @@ def not_required_interface
     def exclusive_primary_interface
       if host && primary?
         primaries = host.interfaces.select { |i| i.primary? && i != self }
-        errors.add :primary, _("host already has primary interface") unless primaries.empty?
+        errors.add :primary, _("interface is already set to the host") unless primaries.empty?
```

Comment:
How does it show up in the UI if you create two of a kind? And a minor grammar correction:
```suggestion
        errors.add :primary, _("interface is already set on the host") unless primaries.empty?
```

Reply:
In the UI it looks like this:
![image](https://user-images.githubusercontent.com/42476762/122172534-84df4c00-ce89-11eb-81ea-696df69603f8.png)